sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_file_client.py

```diff
@@ -310,13 +310,16 @@ def _flush_data_options(offset, content_settings=None, retain_uncommitted_data=F
         return options
 
     def flush_data(self, offset,  # type: int
+                   content_settings=None,  # type: Optional[ContentSettings]
```

Comment:
Hi @CasperAlant Thanks for your pull request! 
Good catch that we missed adding doc for content_settings.
Actually you can pass content_settings as a parameter without this pull request. Because content_settings will be parsed from the kwargs see line 289 in this file